package.yml

```diff
@@ -23,7 +23,7 @@ page:
             itemclass: pull-right
 
 requires:
-    redaxo: ^5.4.0
+    redaxo: ^5.7.0
```

Comment:
Kannst du kurz festhalten, warum 5.7.0 vorausgesetzt wird?

Reply:
das Addon wirft ansonsten Fehler im Frontend bedingt durch den Aufruf der Methode addGlobalCreateFields() in der Klasse lig/Url/UrlManagerSql.php welche vor 5.7.0 nicht angemeldete User nicht verarbeiten konnte